docs/UsersGuide/source/BuildRunSRW.rst

```diff
@@ -0,0 +1,883 @@
+
+.. code-block:: console
+
+   module use <path_to_rocoto_package>
+   module load rocoto
+
+The commands for specific Level 1 platforms are described here: 
+
+Cheyenne:
+
+.. code-block:: console
+
+   module use -a /glade/p/ral/jntp/UFS_SRW_app/modules/
+   module load rocoto
+
+Hera and Jet:
+
+.. code-block:: console
+
+   module purge
+   module load rocoto
+
+Orion:
+
+.. code-block:: console
+
+   module purge
+   module load contrib rocoto
+
+Gaea:
+
+.. code-block:: console
+
+   module use /lustre/f2/pdata/esrl/gsd/contrib/modulefiles
+   module load rocoto/1.3.3
+
+WCOSS_DELL_P3:
+
+.. code-block:: console
+
+   module purge
+   module load lsf/10.1
+   module use /gpfs/dell3/usrx/local/dev/emc_rocoto/modulefiles/
+   module load ruby/2.5.1 rocoto/1.2.4
+
+WCOSS_CRAY:
+
+.. code-block:: console
+
+   module purge
+   module load xt-lsfhpc/9.1.3
+   module use -a /usrx/local/emc_rocoto/modulefiles
+   module load rocoto/1.2.4
+
+
+Run the Rocoto Workflow
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+After loading Rocoto, call ``rocotorun`` from the experiment directory to launch the workflow tasks. This will start any tasks that do not have a dependency. As the workflow progresses through its stages, ``rocotostat`` will show the state of each task and allow users to monitor progress: 
+
+.. code-block:: console
+
+   cd $EXPTDIR
+   rocotorun -w FV3LAM_wflow.xml -d FV3LAM_wflow.db -v 10
+   rocotostat -w FV3LAM_wflow.xml -d FV3LAM_wflow.db -v 10
+
+The ``rocotorun`` and ``rocotostat`` commands above will need to be resubmitted regularly and repeatedly until the experiment is finished. In part, this is to avoid having the system time out. This also ensures that when one task ends, tasks dependent on it will run as soon as possible, and ``rocotostat`` will capture the new progress. 
+
+If the experiment fails, the ``rocotostat`` command will indicate which task failed. Users can look at the log file in the ``log`` subdirectory for the failed task to determine what caused the failure. For example, if the ``make_grid`` task failed, users can open the ``make_grid.log`` file to see what caused the problem: 
+
+.. code-block:: console
+
+   cd $EXPTDIR/log
+   vi make_grid.log
+
+.. note::
+   
+   If users have the `Slurm workload manager <https://slurm.schedmd.com/documentation.html>`_ on their system, they can run the ``squeue`` command in lieu of ``rocotostat`` to check what jobs are currently running. 
+
+.. _Automate:
+
+Automated Option
+----------------------
+For automatic resubmission of the workflow at regular intervals (e.g., every minute), the user can add a crontab entry using the ``crontab -e`` command. As mentioned in :numref:`Section %s <GenerateWorkflow>`, the last line of output from ``./generate_FV3LAM_wflow.sh`` (starting with ``*/1 * * * *``), can be pasted into the crontab file. It can also be found in the ``$EXPTDIR/log.generate_FV3LAM_wflow`` file. The crontab entry should resemble the following: 
+
```

Comment:
I might suggest "every three minutes", since NOAA HPC admins discourage "every minute" due to load problems.  */1 * * * * would need to be changed below as well.

Reply:
Do the NOAA HPC admins handle NOAA Cloud/ParallelWorks as well? NOAA Cloud platforms automatically generate a Rocoto command with */1 * * * * , but I noticed that on Orion it was a */3 command instead. I can add a note to the docs if it's different between Cloud and non-cloud HPC systems.

Reply:
I'm not familiar with how NOAA is managing the cloud/parallelworks systems, but I would assume they would discourage requests to the scheduler from being submitted too often on those systems as well.  @mark-a-potts would probably be able to comment on those platforms.

Reply:
I changed it to every minute on the cloud because the compute nodes don't remain idle for very long before they get shut down after the last job. And, if the compute node shuts down and has to be brought up again, that takes 5-10 minutes. So, for noaacloud, I would recommend no more than every 2 minutes to make sure you don't get stuck with that issue. Any other platform, there is no need to set cron to check so frequently.